conans/model/cpp_package.py

```diff
@@ -0,0 +1,72 @@
+class CppPackage(object):
+    """
+    models conan_package.json, serializable object
+    """
+
+    DEFAULT_FILENAME = "cpp_package.json"
+
+    def __init__(self):
+        self.components = dict()
+
+    @classmethod
+    def load(cls, filename=DEFAULT_FILENAME):
+        """
+        loads package model into memory from the conan_package.json file
+        :param filename: path to the conan_package.json
+        :return: a new instance of ConanPackage
+        """
+        import json
```

Comment:
More local imports, I didn't mark them all, please do not use local imports unless necessary to avoid cycles.

Reply:
removed them all